mailet/standard/src/main/java/org/apache/james/transport/matchers/AtMost.java

```diff
@@ -33,14 +35,16 @@
 import org.apache.mailet.base.GenericMatcher;
 import org.apache.mailet.base.MailetUtil;
 
+import com.google.common.base.Preconditions;
+import com.google.common.base.Splitter;
 import com.google.common.collect.ImmutableList;
 
 /**
- * Checks that a mail did at most X tries on a specific operation.
- *
- * If no retries have been performed previously, it sets up an attribute `AT_MOST_TRIES`
+ * Checks that a mail did at most X retries on a specific operation.
+ * <p>
+ * If no retries have been performed previously, it sets up an attribute `AT_MOST_RETRIES`
```

Comment:
rebase issue? That's the old description

You should though modify this javadoc to include that new attribute name parameter